src/openassetio-python/cmodule/src/_openassetio.hpp

```diff
@@ -116,7 +116,7 @@ void registerEntityReference(const py::module& mod);
 void registerBatchElementError(py::module& mod);
 
 // Register errors, linking exceptions to/from Python.
-void registerErrors(py::module& mod);
+void registerErrors(const py::module& mod);
```

Comment:
[consider] If we're going for consistency, should we do the same to `BatchElementError` whilst we're here?

Reply:
Yeah if possible :+1: - though should that be done as part of the other issue?

Reply:
Might as well, it's already done on that branch.